product_contract/models/sale_order_line.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    contract_id = fields.Many2one(
+        'account.analytic.account',
```

Comment:
No need to change, but going forward I prefer that you statically define all keys in field definitions. `comodel_name`